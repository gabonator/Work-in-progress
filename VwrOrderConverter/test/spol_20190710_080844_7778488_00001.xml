<?xml version="1.0" encoding="UTF-8"?><Envelope xmlns="http://schemas.microsoft.com/dynamics/2008/01/documents/Message"><Header><MessageId>{F9EEFC10-7C6E-437D-96D7-A080E80B3FE9}</MessageId><SourceEndpoint>Spolek</SourceEndpoint><DestinationEndpoint>AIFPurch_D71956</DestinationEndpoint><Action>PURCHORDER</Action></Header><Body><MessageParts xmlns="http://schemas.microsoft.com/dynamics/2008/01/documents/Message"><PurchaseRequisition xmlns="http://schemas.microsoft.com/dynamics/2008/01/documents/PurchaseRequisition"><DocPurpose>Original</DocPurpose><SenderId>spol</SenderId><VendPurchOrderJour class="entity"><_DocumentHash>e18d7c74f9cc95c1d1cd1f82e35c7d41</_DocumentHash><AddressCode>00011789</AddressCode><Amount>24894.92</Amount><AmountPurchaseOrder>Yes</AmountPurchaseOrder><CountryRegionId>CZ</CountryRegionId><CurrencyCode>CZK</CurrencyCode><DeliveryAddress>Kekulova/Sklářská
400 32  Ústí nad Labem</DeliveryAddress><DeliveryCity>Ústí nad Labem</DeliveryCity><DeliveryCountryRegionId>CZ</DeliveryCountryRegionId><DeliveryDate>2019-04-05</DeliveryDate><DeliveryName>Spolek pro chemickou a hutní výrobu, akciová společnost</DeliveryName><DeliveryState>42</DeliveryState><DeliveryStreet>Kekulova/Sklářská</DeliveryStreet><DeliveryType>None</DeliveryType><DeliveryZipCode>400 32</DeliveryZipCode><Dimension><element /><element>10905</element><element /><element>SPO</element></Dimension><ExchRate>100.000000000000</ExchRate><InclTax>No</InclTax><InterCompanyPosted>No</InterCompanyPosted><InvoiceAccount>D71956</InvoiceAccount><ItemBuyerGroupId>P02</ItemBuyerGroupId><LanguageId>CS</LanguageId><OrderAccount>D71956</OrderAccount><ParmId>7962928</ParmId><Payment>90D</Payment><PurchaseOrderId>SP19-04370</PurchaseOrderId><PurchId>N19-04644</PurchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchOrderDocNum>N19-04644-1</PurchOrderDocNum><Qty>309.000</Qty><RecId>5638162175</RecId><RecVersion>532927772</RecVersion><Requisitioner>06736</Requisitioner><SalesOrderbalance>20574.31</SalesOrderbalance><Status>Created</Status><SumTax>4320.61</SumTax><Triangulation>No</Triangulation><VendGroup>TUZ</VendGroup><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-18</DeliveryDate><Dimension><element /><element>10903</element><element /><element>SPO</element></Dimension><EmplEDIEmail>Dvorakova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Dvořáková Alena</EmplEDIEmplName><EmplEDIOffice>10903</EmplEDIOffice><EmplEDIPhone>+420477162015</EmplEDIPhone><ExternalItemId>612-3609</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>5.00</InventQty><InventTransId>ID03536728</InventTransId><ItemId>0048534</ItemId><LineAmount>787.50</LineAmount><LineNum>1.0000000000</LineNum><Name>Pipeta Pasteurova PE, Brand, bez graduace (bal.500ks) objem 4ml, tenká špička</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>157.50</PurchPrice><PurchUnit>ks</PurchUnit><Qty>5.000</Qty><RecId>5638367510</RecId><RecVersion>894739885</RecVersion><TaxAmount>165.38</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536728</InventTransId><Qty>5.00</Qty><RecId>5641371466</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-18</DeliveryDate><Dimension><element /><element>10903</element><element /><element>SPO</element></Dimension><EmplEDIEmail>Dvorakova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Dvořáková Alena</EmplEDIEmplName><EmplEDIOffice>10903</EmplEDIOffice><EmplEDIPhone>+420477162015</EmplEDIPhone><ExternalItemId>BRND747750</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>5.00</InventQty><InventTransId>ID03536729</InventTransId><ItemId>0048534</ItemId><LineAmount>2454.50</LineAmount><LineNum>2.0000000000</LineNum><Name>Pipeta Pasteurova PE, Brand, bez graduace (bal.500ks)</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>490.90</PurchPrice><PurchUnit>ks</PurchUnit><Qty>5.000</Qty><RecId>5638367511</RecId><RecVersion>964881282</RecVersion><TaxAmount>515.45</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536729</InventTransId><Qty>5.00</Qty><RecId>5641371467</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-22</DeliveryDate><Dimension><element /><element>10912</element><element /><element>SPO</element></Dimension><EmplEDIEmail>drasnarova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Drašnarová Jitka</EmplEDIEmplName><EmplEDIOffice>10912</EmplEDIOffice><EmplEDIPhone>+420477162629, 2630</EmplEDIPhone><ExternalItemId>GLAS071.202.20</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>20.00</InventQty><InventTransId>ID03536730</InventTransId><ItemId>0027430</ItemId><LineAmount>2281.20</LineAmount><LineNum>3.0000000000</LineNum><Name>Baňka kuželová ERLEN. s NZ 29/32, 250ml</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>114.06</PurchPrice><PurchUnit>ks</PurchUnit><Qty>20.000</Qty><RecId>5638367512</RecId><RecVersion>933219359</RecVersion><TaxAmount>479.05</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536730</InventTransId><Qty>20.00</Qty><RecId>5641371468</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-22</DeliveryDate><Dimension><element /><element>10912</element><element /><element>SPO</element></Dimension><EmplEDIEmail>drasnarova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Drašnarová Jitka</EmplEDIEmplName><EmplEDIOffice>10912</EmplEDIOffice><EmplEDIPhone>+420477162629, 2630</EmplEDIPhone><ExternalItemId>GLAS075.202.02</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>10.00</InventQty><InventTransId>ID03536731</InventTransId><ItemId>0027433</ItemId><LineAmount>934.10</LineAmount><LineNum>4.0000000000</LineNum><Name>Baňka Erlen. se závitem GL25 vč.uzávěru, 100 ml</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>93.41</PurchPrice><PurchUnit>ks</PurchUnit><Qty>10.000</Qty><RecId>5638367513</RecId><RecVersion>2018279499</RecVersion><TaxAmount>196.16</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536731</InventTransId><Qty>10.00</Qty><RecId>5641371469</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-22</DeliveryDate><Dimension><element /><element>10912</element><element /><element>SPO</element></Dimension><EmplEDIEmail>drasnarova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Drašnarová Jitka</EmplEDIEmplName><EmplEDIOffice>10912</EmplEDIOffice><EmplEDIPhone>+420477162629, 2630</EmplEDIPhone><ExternalItemId>VTRB632411103250</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>10.00</InventQty><InventTransId>ID03536732</InventTransId><ItemId>0027256</ItemId><LineAmount>990.00</LineAmount><LineNum>5.0000000000</LineNum><Name>Baňka titrační s plochým dnem širokohrdlá, 250ml, vyhnutý okraj</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>99.00</PurchPrice><PurchUnit>ks</PurchUnit><Qty>10.000</Qty><RecId>5638367514</RecId><RecVersion>1455864777</RecVersion><TaxAmount>207.90</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536732</InventTransId><Qty>10.00</Qty><RecId>5641371470</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-22</DeliveryDate><Dimension><element /><element>10912</element><element /><element>SPO</element></Dimension><EmplEDIEmail>drasnarova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Drašnarová Jitka</EmplEDIEmplName><EmplEDIOffice>10912</EmplEDIOffice><EmplEDIPhone>+420477162629, 2630</EmplEDIPhone><ExternalItemId>GLAS071.202.14</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>10.00</InventQty><InventTransId>ID03536733</InventTransId><ItemId>0048483</ItemId><LineAmount>966.20</LineAmount><LineNum>6.0000000000</LineNum><Name>Baňka kuželová ERLEN. s NZ  29/32, 100 ml</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>96.62</PurchPrice><PurchUnit>ks</PurchUnit><Qty>10.000</Qty><RecId>5638367515</RecId><RecVersion>1535207553</RecVersion><TaxAmount>202.90</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536733</InventTransId><Qty>10.00</Qty><RecId>5641371471</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-25</DeliveryDate><Dimension><element /><element>10903</element><element /><element>SPO</element></Dimension><EmplEDIEmail>Dvorakova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Dvořáková Alena</EmplEDIEmplName><EmplEDIOffice>10903</EmplEDIOffice><EmplEDIPhone>+420477162015</EmplEDIPhone><ExternalItemId>VITU21500</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>120.00</InventQty><InventTransId>ID03536734</InventTransId><ItemId>00000446</ItemId><LineAmount>3210.00</LineAmount><LineNum>7.0000000000</LineNum><Name>Kelímek bílý PP se šr. uz. 625ml (500g)</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>26.75</PurchPrice><PurchUnit>ks</PurchUnit><Qty>120.000</Qty><RecId>5638367516</RecId><RecVersion>874939613</RecVersion><TaxAmount>674.10</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536734</InventTransId><Qty>120.00</Qty><RecId>5641371472</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-25</DeliveryDate><Dimension><element /><element>10903</element><element /><element>SPO</element></Dimension><EmplEDIEmail>Dvorakova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Dvořáková Alena</EmplEDIEmplName><EmplEDIOffice>10903</EmplEDIOffice><EmplEDIPhone>+420477162015</EmplEDIPhone><ExternalItemId>SIMPB700-100/100ks</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>5.00</InventQty><InventTransId>ID03536735</InventTransId><ItemId>0027316</ItemId><LineAmount>4072.95</LineAmount><LineNum>8.0000000000</LineNum><Name>Kádinka PP tři-výtoková 100 ml, obj.č.212-9302, bal.100ks</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>814.59</PurchPrice><PurchUnit>ks</PurchUnit><Qty>5.000</Qty><RecId>5638367517</RecId><RecVersion>522340851</RecVersion><TaxAmount>855.32</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536735</InventTransId><Qty>5.00</Qty><RecId>5641371473</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-25</DeliveryDate><Dimension><element /><element>10903</element><element /><element>SPO</element></Dimension><EmplEDIEmail>Dvorakova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Dvořáková Alena</EmplEDIEmplName><EmplEDIOffice>10903</EmplEDIOffice><EmplEDIPhone>+420477162015</EmplEDIPhone><ExternalItemId>IBSA040-228</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>112.00</InventQty><InventTransId>ID03536736</InventTransId><ItemId>0027996</ItemId><LineAmount>1784.16</LineAmount><LineNum>9.0000000000</LineNum><Name>Lékovka bílá s uzávěrem 130ml, širokohrdlá, obj.č. 635 038 040 228 (balení 56ks)</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>15.93</PurchPrice><PurchUnit>ks</PurchUnit><Qty>112.000</Qty><RecId>5638367518</RecId><RecVersion>98969800</RecVersion><TaxAmount>374.67</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536736</InventTransId><Qty>112.00</Qty><RecId>5641371474</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-26</DeliveryDate><Dimension><element /><element>10912</element><element /><element>SPO</element></Dimension><EmplEDIEmail>drasnarova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Drašnarová Jitka</EmplEDIEmplName><EmplEDIOffice>10912</EmplEDIOffice><EmplEDIPhone>+420477162629, 2630</EmplEDIPhone><ExternalItemId>181-0296</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>1.00</InventQty><InventTransId>ID03536737</InventTransId><ItemId>0027371</ItemId><LineAmount>2699.00</LineAmount><LineNum>10.0000000000</LineNum><Name>Set rychlospojek pro připojení VHC lahve č.adaptéru a přívodní hadice pro aspir.systém Vacuubrand</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>2699.00</PurchPrice><PurchUnit>ks</PurchUnit><Qty>1.000</Qty><RecId>5638367519</RecId><RecVersion>1786878992</RecVersion><TaxAmount>566.79</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536737</InventTransId><Qty>1.00</Qty><RecId>5641371475</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-03-27</DeliveryDate><Dimension><element /><element>10922</element><element /><element>SPO</element></Dimension><EmplEDIEmail>Dvorakova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Dvořáková Alena</EmplEDIEmplName><EmplEDIOffice>10922</EmplEDIOffice><EmplEDIPhone>+420477162015</EmplEDIPhone><ExternalItemId>282224416428</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>10.00</InventQty><InventTransId>ID03536738</InventTransId><ItemId>0026705</ItemId><LineAmount>185.10</LineAmount><LineNum>11.0000000000</LineNum><Name>Kartáč na lahve prům.50 mm s vějířkem, držadlo, silon</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>18.51</PurchPrice><PurchUnit>ks</PurchUnit><Qty>10.000</Qty><RecId>5638367520</RecId><RecVersion>156290960</RecVersion><TaxAmount>38.87</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536738</InventTransId><Qty>10.00</Qty><RecId>5641371476</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><VendPurchOrderTrans class="entity"><CurrencyCode>CZK</CurrencyCode><DeliveryDate>2019-04-05</DeliveryDate><Dimension><element /><element>10905</element><element /><element>SPO</element></Dimension><EmplEDIEmail>Jiroutova@spolchemie.cz</EmplEDIEmail><EmplEDIEmplName>Jiroutová Romana</EmplEDIEmplName><EmplEDIOffice>10905</EmplEDIOffice><EmplEDIPhone>+420477162156</EmplEDIPhone><ExternalItemId>PLUM4604</ExternalItemId><InventDimId>DIM000017</InventDimId><InventQty>1.00</InventQty><InventTransId>ID03536739</InventTransId><ItemId>0032021</ItemId><LineAmount>209.60</LineAmount><LineNum>12.0000000000</LineNum><Name>Roztok na výplach očí s 0,9% kuchyň.soli PLUM 500ml</Name><OrigPurchId>N19-04644</OrigPurchId><PriceUnit>1.00</PriceUnit><PurchaseOrderId>SP19-04370</PurchaseOrderId><purchId>N19-04644</purchId><PurchOrderDate>2019-03-29</PurchOrderDate><PurchPrice>209.60</PurchPrice><PurchUnit>ks</PurchUnit><Qty>1.000</Qty><RecId>5638367521</RecId><RecVersion>552059871</RecVersion><TaxAmount>44.02</TaxAmount><TaxGroup>CZN</TaxGroup><TaxItemGroup>PZ</TaxItemGroup><InventReportDimHistory class="entity"><InventDimId>DIM000017</InventDimId><InventTransId>ID03536739</InventTransId><Qty>1.00</Qty><RecId>5641371477</RecId><RecVersion>1</RecVersion><TransactionLogType>PurchPurchaseOrder</TransactionLogType><TransRefId>SP19-04370</TransRefId><InventDim class="entity"><inventDimId>DIM000017</inventDimId><InventLocationId>223</InventLocationId><RecId>5637156177</RecId><RecVersion>1</RecVersion></InventDim></InventReportDimHistory></VendPurchOrderTrans><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks><FormLetterRemarks class="entity"><LanguageId>CS</LanguageId></FormLetterRemarks></VendPurchOrderJour></PurchaseRequisition></MessageParts></Body></Envelope>