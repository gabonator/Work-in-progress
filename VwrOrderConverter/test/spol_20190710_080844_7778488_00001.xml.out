BEGIN;N19-04644-1;Spolek pro chemickou a hutní výrobu, akciová společnost;VWR;ORDERS;20190329;102511;HEAD;N19-04644-1;20190329;;CZK;SOLDTO;26000135;Spolek pro chemickou a hutní výrobu, akciová společnostSHIPTO;26000135;Spolek pro chemickou a hutní výrobu, akciová společnost;;;;Kekulova/Sklářská;;;;;Ústí nad Labem;400 32;CZ;korinkova@spolchemie.cz;477 164 632LINE;000001;;612-3609.;Pipeta Pasteurova PE, Brand, bez graduace (bal.500ks) objem 4ml, tenká špička;5.000;;20190329;157.5;787.50LINE;000002;;BRND747750.;Pipeta Pasteurova PE, Brand, bez graduace (bal.500ks);5.000;;20190329;490.9;2454.50LINE;000003;;GLAS071.202.20.;Baňka kuželová ERLEN. s NZ 29/32, 250ml;20.000;;20190329;114.06;2281.20LINE;000004;;GLAS075.202.02.;Baňka Erlen. se závitem GL25 vč.uzávěru, 100 ml;10.000;;20190329;93.41;934.10LINE;000005;;VTRB632411103250.;Baňka titrační s plochým dnem širokohrdlá, 250ml, vyhnutý okraj;10.000;;20190329;99;990.00LINE;000006;;GLAS071.202.14.;Baňka kuželová ERLEN. s NZ  29/32, 100 ml;10.000;;20190329;96.62;966.20LINE;000007;;VITU21500.;Kelímek bílý PP se šr. uz. 625ml (500g);120.000;;20190329;26.75;3210.00LINE;000008;;SIMPB700-100/100ks.;Kádinka PP tři-výtoková 100 ml, obj.č.212-9302, bal.100ks;5.000;;20190329;814.59;4072.95LINE;000009;;IBSA040-228.;Lékovka bílá s uzávěrem 130ml, širokohrdlá, obj.č. 635 038 040 228 (balení 56ks);112.000;;20190329;15.93;1784.16LINE;000010;;181-0296.;Set rychlospojek pro připojení VHC lahve č.adaptéru a přívodní hadice pro aspir.systém Vacuubrand;1.000;;20190329;2699;2699.00LINE;000011;;282224416428.;Kartáč na lahve prům.50 mm s vějířkem, držadlo, silon;10.000;;20190329;18.51;185.10LINE;000012;;PLUM4604.;Roztok na výplach očí s 0,9% kuchyň.soli PLUM 500ml;1.000;;20190329;209.6;209.60END;N19-04644-1